JitsiConference.js

```diff
@@ -147,7 +147,7 @@ export default function JitsiConference(options) {
     this.connection = options.connection;
     this.xmpp = this.connection?.xmpp;
 
-    if (this.xmpp.isRoomCreated(options.name, options.customDomain)) {
```

Comment:
Why do we need this change?

Reply:
This was causing problems ... let me try to remember it was few months ago ...

Reply:
No idea, ok I will drop the commit when force pushing the other changes.

Reply:
We need to pass xmpp to Moderator, and in XmppConnection we pass just options no instance of other objects.
We need the xmpp.eventEmitter in Moderator.